geode-core/src/test/java/org/apache/geode/distributed/internal/membership/MembershipDependenciesJUnitTest.java

```diff
@@ -160,8 +167,24 @@
               .or(type(DistributedMember.class))
               .or(type(MembershipView.class))
 
-              .or(type(GMSMembershipManager.class))
               .or(type(ClusterDistributionManager.class))
 
+              // TODO: GMSMembershipMannager move brought these within...
+              .or(type(SystemTimer.class))
+              .or(type(SystemTimer.SystemTimerTask.class))
+              .or(type(Assert.class))
+              .or(type(InternalLocator.class))
+              .or(type(GMSMessageAdapter.class))
+              .or(type(GMSLocatorAdapter.class))
+              .or(type(QuorumChecker.class))
+              .or(type(LogMarker.class))
+              .or(type(SystemFailure.class))
+              .or(type(MembershipTestHook.class))
+              .or(type(GMSQuorumChecker.class))
+              .or(type(GMSQuorumCheckerAdapter.class))
+              .or(type(InternalDistributedSystem.class))
+              .or(type(MembershipView.class))
+              .or(type(LocalViewMessage.class))
+
```

Comment:
This is a big step backward. I feel like we should have tickets covering these new dependencies before we merge this PR.

Reply:
The plan was to fix/resolve these under the same ticket in the next PR